modules/datadog-slo/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "region" {
```

Comment:
is this variable in use?

Reply:
https://github.com/cloudposse/terraform-aws-components/pull/372/files#diff-5ea1ebc8209e3a63552e52bebb01db3bc73482ff110229e554bbd109aa97830bR8

Reply:
Yes